### PR TITLE
[FOOD-84] [Admin] Arreglar los espaciados para el navbar del admin

### PR DESCRIPTION
**Pull request by Pericena**
## Jira ticket
https://locotoinnovations.atlassian.net/browse/FOOD-84
diagrama: https://drive.google.com/file/d/1OJpGfCZIYBmmsZI2oB7r3qxdeG2i77Qr/view?usp=sharing
## Context

Significant improvements have been made to the navbar to ensure responsiveness across a variety of devices including mobile, tablet and desktop.

Also included
**Notifications:** Improved the notifications button with a drop-down menu.
**User Menu:** Added an avatar with a menu offering options such as profile, settings, and logout.

## Screenshot and videos
**escritorio**
_dark_
image
_dark User_
image

_Light_
image

_Light User_
image

_Light Nitifications_
image

**table**
_dark_
image
_dark User_
image

_Light_
image
_Light User_
image

**mobile**
_dark_
image
_dark User_
image

_Light_
image
_Light User_
image

### CODE REVIEW

File: src/components/admin/headernav/NexHeaderSearchMobil.astro [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<div class="px-4 ">
```
Comment (electricpants01) [inline src/components/admin/headernav/NexHeaderSearchMobil.astro:1]: rename this to `NexHEaderSearchMobile.astro`

**Comment by electricpants01** [comment]
Al tocar la imagen del usuario tenemos: "Profile", "Settings" y "Log out", lo mismo deberia mostrarse en mobile


| Web  | Mobile  |
|---|---|
| image  | image  |

File: src/components/admin/headernav/NexHeaderSearchMobil.astro (hunk 1 above)
Comment (Pericena) [reply src/components/admin/headernav/NexHeaderSearchMobil.astro]: 7e582da7cf9890b8ea8d4b0c55fbbfff5aa05380

[FOOD-84] [style]:Renamed the file NexHeaderSearchMobil.astro to NexHeaderSearchMobile.astro

File: src/components/client/home/headernav/NexHeaderMain.astro [hunk 2]
```diff
@@ -1,17 +1,77 @@
...
+      <NexHeaderUser />
+    </div>
+  </div>
+  <!-- Mobile menu -->
+
+  <div
+    id="mobile-menu"
+    class="mt-4 hidden w-full dmd:hidden"
+  >
+    <NexHeaderSearchMobil />
```
Comment (electricpants01) [inline src/components/client/home/headernav/NexHeaderMain.astro:51]: renombrar a `NexHeaderSearchMobil`

File: src/components/client/home/headernav/NexHeaderNotifications.astro [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+>
+  <div
+    class="block rounded-t-lg bg-adminLightSecondary px-4 py-2 text-center font-medium text-adminLightTextColor dark:bg-adminDarkSecondary dark:text-adminDarkTextColor"
+  >
+    Notifications
+  </div>
+  <div class="divide-y divide-adminLightPrimary dark:divide-adminDarkPrimary">
+    <a href="#" class="flex px-4 py-3">
+      <div class="relative flex-shrink-0">
+        <!-- Imagen del usuario -->
```
Comment (electricpants01) [inline src/components/client/home/headernav/NexHeaderNotifications.astro:51]: Seria bueno mover un `notificationItem` dentro de su propio componente y luego en este archivo hacer un `forEach`

File: src/components/client/home/headernav/NexHeaderThemeButton.astro [hunk 4]
```diff
@@ -0,0 +1,109 @@
+<div class="flex items-center justify-center space-x-2">
+  <div class="relative flex items-center">
```
Comment (electricpants01) [inline src/components/client/home/headernav/NexHeaderThemeButton.astro:2]: Se va continuar trabajando en el ticket de Light/Dark mode para el cliente

**Review by electricpants01 — APPROVED** [review]

**Pull request merged**